input/docs/getting-started/upgrade.md

```diff
@@ -3,6 +3,28 @@ Title: Upgrade instructions
 Description: Upgrade instructions between different versions of Cake
 ---
 
+If you are using the [.NET tool as we recommend](/docs/running-builds/runners/dotnet-tool), all you need to do is to update the Cake .NET tool:
+
+```powershell
+# If you are using the .NET tool as a local tool for the repository
+dotnet tool install --local Cake.Tool
+
+# or
+
+# If you are using the .NET tool as a global tool for the user
+dotnet tool install --global Cake.Tool
+```
+
+### Note about Cake.DotNetTool.Module
+
+If you use `Cake.DotNetTool.Module` already on your builds, you should either:
```

Comment:
An unordered list with a number first looks strange. As it is an either/or choice an unordered list should work fine:

```
If you use `Cake.DotNetTool.Module` already on your builds do one of the following:

- Upgrade Cake to version 1.1.0, and _remove_ the `Cake.DotNetTool.Module` from your build script (as it's no longer needed).
- Pin `Cake.DotNetTool.Module` to a version compatible with the Cake version that you use. `Cake.DotNetTool.Module` version 1.0.1 is the last version compatible with Cake 1.0.0. Starting with this release, `Cake.DotNetTool.Module` will always be released together with Cake and will only be compatible with the current release.
```

Reply:
:white_check_mark: